expression/integration_test.go

```diff
@@ -10382,6 +10382,17 @@ func (s *testIntegrationSuite) TestLastInsertId(c *C) {
 	tk.MustQuery("SELECT @@last_insert_id, LAST_INSERT_ID()").Check(testkit.Rows("3 3"))
 }
 
+func (s *testIntegrationSuite) TestTimestamp(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec(`use test;`)
+	tk.MustQuery("SELECT @@timestamp;")
+	tk.MustExec("set @@timestamp = 12345;")
+	tk.MustQuery("SELECT @@timestamp;").Check(testkit.Rows("12345"))
+	tk.MustExec("set @@timestamp = default;")
+	tk.MustQuery("SELECT @@timestamp;")
+
```

Comment:
Please add a test `set @@timestamp=0;select now();` which should also reset the timestamp to default (MySQL does). I think the code needs to be updated for supporting this as well.